test/Microsoft.SqlTools.ServiceLayer.Test/LanguageServer/LanguageServiceTests.cs

```diff
@@ -308,10 +312,21 @@ public void AutoCompleteFindCompletions()
                 }
             };
 
-            // If we have a valid ScriptParseInfo and the SQL has already been parsed
+            // If we have a valid ScriptParseInfo
             var service = LanguageService.Instance;
-            ScriptParseInfo parseInfo = service.GetScriptParseInfo(scriptFile.ClientFilePath);
-            await service.UpdateLanguageServiceOnConnection(connInfo);
+            ScriptParseInfo scriptInfo = service.GetScriptParseInfo(scriptFile.ClientFilePath);
+
+            // And we parse the SQL 
+            // (this will perform the same operations as UpdateLanguageServiceOnConnection, but without
+            // the ServiceHost.Instance.SendEvent, which will throw in a test environment) 
+            var bindingQueue = LanguageService.Instance.BindingQueue;
```

Comment:
ideally we don't duplicate product code into the tests and figure out how to prevent the product code from failing, for example, may we should add a "send telemetry event" helper method" that try\catch and ignores errors...this is a pretty simple snippet so it's probably fine in this case, but better to avoid if possible.